### DISCUSSION TITLE
Will the new RX 7900 XTX and RX 7900 XT series of GPUs be added?

### DISCUSSION THREAD

**Discussion by tempestbk** [jef/streetmerchant#3110] [Ideas and feature requests]
I know they just launched today. Just curious if these will be added at some point?

thank you.

**Comment by Mr-Tech-13** [comment]
I will update my PR tomorrow, and if jef accepts, they will be added

**Reply by tempestbk** [reply]
Thank you!

**Reply by Mr-Tech-13** [reply]
updated
